polymorphia-backend/src/main/java/com/agh/polymorphia_backend/model/course/EvolutionStage.java

```diff
@@ -0,0 +1,29 @@
+package com.agh.polymorphia_backend.model.course;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "evolution_stages")
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class EvolutionStage {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String description;
+    private int minXp;
+    @Column(precision = 2, scale = 1)
+    private BigDecimal grade;
+    private String imageUrl;
+    private String imageWithoutBgUrl;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "course_id")
+    private Course course;
```

Comment:
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    private String name;
    
    private String description;
    
    private int minXp;
    
    @Column(precision = 2, scale = 1)
    private BigDecimal grade;
    
    private String imageUrl;
    
    private String imageWithoutBgUrl;
    
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "course_id")
    private Course course;
```

But that's just my preference, open for discussion.

Reply:
Is it just formatting? If so, I appreciate the change